src/voprf.rs

```diff
@@ -662,6 +697,59 @@ pub struct VerifiableServerEvaluateResult<G: Group, H: BlockSizeUser + Digest +
     pub proof: Proof<G, H>,
 }
 
+/// Contains prepared [`EvaluationElement`]s by a verifiable server batch
+/// evaluate preparation.
+pub struct PreparedEvaluationElement<G: Group, H: BlockSizeUser + Digest + FixedOutputReset>(
+    EvaluationElement<G, H>,
+);
+
+/// Contains the prepared `t` by a verifiable server batch evaluate preparation.
+#[derive(DeriveWhere)]
+#[derive_where(Zeroize(drop))]
+pub struct PreparedT<G: Group, H: BlockSizeUser + Digest + FixedOutputReset> {
```

Comment:
I couldn't find a better name, any suggestions?

Reply:
Can we change this to `PreparedTScalar`?

Reply:
Renamed to `PreparedTscalar`, see https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case.